libqtile/widget/statusnotifier.py

```diff
@@ -563,7 +563,7 @@ class StatusNotifier(base._Widget):
     defaults = [
         ('icon_size', 16, 'Icon width'),
         ('icon_theme', None, 'Name of theme to use for app icons'),
-        ('padding', 5, 'Padding between icons'),
+        ('padding', 3, 'Padding between icons'),
```

Comment:
This value wasn't overriding the `widget_defaults` value so I've made this match that value. Users shouldn't notice any difference.